analyses/immune-deconv/02-summary-plots.R

```diff
@@ -29,13 +29,9 @@ deconvout <- opt$input
 output <- opt$output
 load(deconvout) 
 
-# if short histology is Medulloblastoma or ATRT, use them as broad histology
```

Comment:
I didn't think this was relevant anymore so I deleted this step completely, but someone should let me know if something like this is still needed.

Reply:
confirmed this is ok to delete with the switch to harmonized dx